### PR TITLE
Update on-boarding documentation

### PR DESCRIPTION
**Pull request by agarzola**
## Description

This PR updates the on-boarding documentation to provide additional instructions that may help developers as they spin up this project for the first time. I based these changes off of lessons learned while trying to set this up locally for the first time, and then went through the steps again with a clean slate to be sure everything works as expected.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,16 +9,28 @@ Drupal backend.
...
 
-The easiest way to get started developing the poets.org frontend magicks is by using [Lando](https://github.com/lando/lando), however if you are feeling pro and desire moar speedz you can also use [yarn](https://yarnpkg.com/en/) straight up.
+### 1. Set Up Your `.env` File
 
-### 1. Using Lando
+After cloning this repo, create a `.env` file at the root of the project, then paste the contents of the _Poets.org Front-end .env File Contents_ note in the Poets.org vault in 1Password. Keep in mind that this file is intentionally ignored in Git, as it holds secrets that should not be tracked in version control.
+
+### 2. Start the API
+
+In order to run the Poets.org site locally, you’ll need to have the `poets-api` application running as well. Follow the instructions in the [`poets-api` README](https://github.com/ChromaticHQ/poets-api) to set up and run the API via Lando. You don’t have to wait until after the API has started to continue setting up the front-end application with the steps below. Just know that, until the API is up and running, the front-end won’t have any content to display.
```
Comment (agarzola) [inline README.md:18]: I realize the link to the `poets-api` repo points to the `ChromaticHQ` org, but I figure since this will be moved next week anyway, I’d rather get ahead of that than have to submit another PR for it after that switch.

**Comment by agarzola** [comment]
@adamzimmermann Assigning this to you so you can merge it after giving it a review.

File: README.md [hunk 2]
```diff
@@ -9,16 +9,28 @@ Drupal backend.
...
+
+### 3. Start the Front-end Application
+
+The easiest way to get started developing the poets.org frontend magicks is by using [Lando](https://github.com/lando/lando), however if you are feeling pro and desire moar speedz you can also use [yarn](https://yarnpkg.com/en/) straight up. Both of these options are detailed below, but the Lando route is preferred, as we have run into issues running the application outside of Lando.
```
Comment (adamzimmermann) [inline README.md:22]:
```suggestion
The easiest way to get started developing the poets.org frontend magicks is by using [Lando](https://github.com/lando/lando). However if you are feeling pro and desire better performance, you can also use [yarn](https://yarnpkg.com/en/) straight up. Both of these options are detailed below, but the Lando route is preferred, as we have run into issues running the application outside of Lando.
```

**Review by adamzimmermann — APPROVED** [review]
